src/compute-client/src/controller.rs

```diff
@@ -545,21 +545,22 @@ where
         replica_id: ReplicaId,
         config: ComputeReplicaConfig,
     ) -> Result<(), ComputeError> {
-        let logging_config = if let Some(interval) = config.logging.interval {
-            let mut sink_logs = BTreeMap::new();
-            for (variant, id) in config.logging.sources {
-                let storage_meta = self.storage.collection(id)?.collection_metadata.clone();
-                sink_logs.insert(variant, (id, storage_meta));
-            }
+        let interval_ns = match config.logging.interval {
+            Some(interval) => interval.as_nanos(),
+            None => 0,
+        };
```

Comment:
I think this is drifting towards us using an idiom to replace clearer information. When the interval is `0` I think this means "continual logging" but we are using it here to mean "disabled", I think. There's probably a tension between "changing Proto is annoying" and "surprising footguns in the future".